resources/js/components/Listing/Listing.vue

```diff
@@ -96,5 +96,24 @@ export default {
             window.history.pushState(window.history.state, '', url)
         },
     },
+
+    methods: {
+        getPositionsQuery() {
+            if (!window.config.category?.entity_id) {
+                return;
+            }
+
+            return {
+                query: {
+                    function_score: {
+                        field_value_factor: {
+                            field: 'positions.'+window.config.category.entity_id,
+                            missing: 0
+                        }
+                    }
```

Comment:
While we're at it, might as well implement the script fix thing? I assume the need for that fix hasn't changed given that this is just a repositioning of the code.